src/Calculator/App.xaml

```diff
@@ -17,6 +17,7 @@
                     <x:Double x:Key="HighContrastStrokeThickness">0</x:Double>
                     <Color x:Key="AltHighColor">#FF000000</Color>
                     <Color x:Key="ChromeMediumLowColor">#FF2B2B2B</Color>
+                    <Color x:Key="AppControlFocusedVariableForegroundText">#FFFFFFFF</Color>
```

Comment:
>AppControlFocusedVariableForegroundText [](start = 34, length = 39)

Does this need to be added to Dark and High Contrast ThemeDictionaries?

Reply:
Actually should have been deleted, gone now.